manifests/profile/python.pp

```diff
@@ -3,14 +3,30 @@
 # @example Basic Usage
 #  include st2::profile::python
 #
-class st2::profile::python {
+# @example Instally python3
+#  class { 'st2':
+#    python_version => '3',
```

Comment:
Can we default this to 3.6? As on one of the OSs there is a default python3, but it's earlier than 3.6, so we still need to use non-system default.

Reply:
Let's get this merged in first, then i can make another PR to change default version. Changing default would require a major version bump because it would potentially upgrade a bunch of people's systems to new versions of python.